src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -454,16 +453,7 @@ private long CreateBackupTask(PeriodicBackup periodicBackup, bool isFullBackup,
                     };
                 }
 
-                if (_serverStore.ConcurrentBackupsSemaphore.Wait(0) == false)
-                {
-                    throw new BackupDelayException(
-                        $"Failed to start Backup Task: '{periodicBackup.Configuration.Name}'. " +
-                        $"The task exceeds the maximum number of concurrent backup tasks configured. " +
-                        $"Current value of Backup.MaxNumberOfConcurrentBackups is: {_serverStore.Configuration.Backup.MaxNumberOfConcurrentBackups:#,#;;0}")
-                    {
-                        DelayPeriod = TimeSpan.FromMinutes(1)
-                    };
-                }
+                var backupCounter = _serverStore.ConcurrentBackupsCounter.StartBackup(periodicBackup.Configuration.Name);
```

Comment:
```suggestion
                using var backupCounter = _serverStore.ConcurrentBackupsCounter.StartBackup(periodicBackup.Configuration.Name);
```

Reply:
Can't dispose here because the backup starts in a background thread.